Dockerfile

```diff
@@ -1,13 +1,13 @@
-ARG PHP_VERSION=7.4
+ARG PHP_VERSION=8.3
```

Comment:
Should we keep it down to 7.4 to ensure backward compat?

Reply:
Maybe 8.1, as the oldest still-supported version?

Reply:
We say "PHP 8" in the README.